ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/console/GraphConsoleView.java

```diff
@@ -119,6 +146,38 @@ private void createUIComponents() {
         graphCanvas = new JPanel(new GridLayout(0, 1));
         consoleTabsPane = new JBTabsPaneImpl(null, SwingConstants.TOP, this);
         consoleTabs = (JBTabsImpl) consoleTabsPane.getTabs();
+
+        consoleTabs.addTabMouseListener(new MouseListener() {
+            @Override
+            public void mouseReleased(MouseEvent e) {
+                if (UIUtil.isCloseClick(e, MouseEvent.MOUSE_RELEASED)) {
+                    final TabInfo info = consoleTabs.findInfo(e);
+                    if (info != null) {
+                        String tabTitle = info.getText();
+                        if (tabTitle.startsWith("Profile") || tabTitle.startsWith("Explain")) {
```

Comment:
If we are making comparisons using strings, then we should put them in constants and reuse constants